src/NVD.hs

```diff
@@ -113,6 +113,24 @@ rebuildDB = do
         , "  published text,"
         , "  modified text)"
         ]
+    execute_ conn $
+      Query $
+      T.unlines
+        [ "CREATE TABLE cpes ("
```

Comment:
I think these fields should become part of the matcher table: The version is often `Nothing` when a version range was specified instead, so without the matcher, this information would be incomplete. See `cpeToMatcher`. It should be changed to always extract one matcher row (now including CPE information). When both `cpeVersion` and `cpeMatcher` are set, it is probably an input error.